apps/nestjs-app/tsconfig.build.json

```diff
@@ -9,11 +9,5 @@
     "test",
     "dist",
     "**/*spec.ts"
-  ],
-  "files": [
-    "src/main.ts"
-  ],
-  "include": [
-    "src/**/*.ts"
```

Comment:
not sure why these were here and not in the tsconfig.

1. doesn't get updated by moon
2. doesn't help the typechecking in the editor